components/prism-sparql.js

```diff
@@ -0,0 +1,17 @@
+Prism.languages.sparql = Prism.languages.extend('turtle', {
+		'variable': {pattern: /(\?|\$)[\w]+/,
+								 greedy: true
```

Comment:
Interesting indentation.

Reply:
Improved the indentation, would look a better now.